HLTrigger/HLTcore/interface/TriggerSummaryProducerAOD.h

```diff
@@ -61,16 +64,34 @@ namespace edm {
 // class declaration
 //
 
-class TriggerSummaryProducerAOD : public edm::EDProducer {
+/// GlobalCache
+struct InputTagHash {
+  std::size_t operator()(const edm::InputTag& inputTag) const {
+    std::hash<std::string> Hash;
+    return Hash(inputTag.encode());
+  }
+};
+
+struct GlobalTags {
```

Comment:
hi Martin,
may I suggest a different name, to avoid confusion with the "GlobalTag" for conditions ?